### ISSUE TITLE
Gatt exchange mtu event not work

### ISSUE THREAD

**Issue by Gearspre** [weliem/bluez_inc#58]
I am trying to use this lib to create gatt server, it works. But no matter how I send the exchange MTU event, its MTU always remains at 517. 
And I find in **application.c line:1035**  has one todo : *deal with the offset & mtu parameter*
So is this a known issue, and is there any simple solution?

**Comment by weliem** [comment]
I have been looking into this but I don't think Bluez has an API for this.

**Comment by Gearspre** [comment]
Yes. I found a link discussing this thing : [Determine ATT MTU size over DBUS](https://lore.kernel.org/all/-JmTPP9qq2ljmysdKs_4nDqzQMZH4NMScqdEFmY7eo4Eu5GDniGqk-lMiLXCo7O1RZgB4BI_UsKZS0KdET69HZksuaRIKE_0mrQJS885K-M=@krauser.org/T/)

Now I have another problem. How can I get the negotiated MTU size from GattCharacteristic? 
A simple solution is get from ReadValue or WriteValue method options. But it's not a good solution. Do you have any idea?

**Comment by weliem** [comment]
Using the MTU from the ReadValue and WriteValue options parameter is totally fine to use. However, that doesn't enforce a MTU. It simply says what the current MTU is....
